data/transactions/logic/eval.go

```diff
@@ -199,7 +199,7 @@ type LedgerForLogic interface {
 	AccountData(addr basics.Address) (ledgercore.AccountData, error)
 	Authorizer(addr basics.Address) (basics.Address, error)
 	Round() basics.Round
-	LatestTimestamp() int64
+	PrevTimestamp() int64
```

Comment:
@jannotti Sanity checking:  It seems the `PrevTimestamp` rename is unrelated to the PR's primary intent.  Is it an intended change?

If yes, I'm noting that there's downstream methods that can be updated to reflect the new name.

Reply:
@jannotti Got it - thanks for the clarification.  Sorry, I had asked prior to seeing the `logicLedger` changes.  Resolving.